src/AttestationRegistry.sol

```diff
@@ -61,7 +64,7 @@ contract AttestationRegistry is OwnableUpgradeable {
   }
 
   /// @custom:oz-upgrades-unsafe-allow constructor
-  constructor() {
+  constructor() ERC1155("NA") {
```

Comment:
We could imagine the URI being a link to the explorer.  
Can you add a `// TODO` so that we think of it?

Reply:
When I used the ERC1155Upgradeable, now it does not need any parameter to constructor